src/Potential/Potential.cpp

```diff
@@ -1,14 +1,16 @@
 #include "Potential.h"
 
```

Comment:
Io un piccolo commento qui lo scriverei, ma nulla di che, giusto per vedere il pelo nell'uovo...

Reply:
You can find everything [here](https://github.com/Scienza/Schroedinger/wiki/Classes). It should be updated, but is useful to understand. Sure everything have to be documented (something like JavaDoc?)

Reply:
Opened issue #45